x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ClientHelper.java

```diff
@@ -195,6 +195,8 @@ private static String maybeRewriteSingleAuthenticationHeaderForVersion(
     public static final String INFERENCE_ORIGIN = "inference";
     public static final String APM_ORIGIN = "apm";
     public static final String OTEL_ORIGIN = "otel";
+    public static final String KIBANA_ORIGIN = "kibana";
```

Comment:
Is there any way we can make sure that we only have one kibana origin (also defined in `KibanaPlugin.java`)?

Reply:
I originally tried to import `KIBANA_ORIGIN` [here](https://github.com/elastic/elasticsearch/blob/9c192684520f4138ddc606e3b26bdf8d010f7153/modules/kibana/src/main/java/org/elasticsearch/kibana/KibanaPlugin.java#L23) so there can be a single origin, but for some reason I can't import `KIBANA_ORIGIN` into this package. I'm at the ends of my Java knowledge (which is very low). Any idea why I can't import this? Is it possibly because `ClientHelper.java` is under an `xpack` while `KibanaPlugin.java` isn't?

Reply:
I think you may need to update the module if we did that. Perhaps add a TODO and tackle that in a followup PR? But we should address it as tech debt if we do this here.